pkgs/desktops/elementary/switchboard-plugs/about/default.nix

```diff
@@ -0,0 +1,56 @@
+{ stdenv, fetchFromGitHub, elementary, substituteAll, meson, ninja, pkgconfig
+, vala, libgee, granite, gtk3, switchboard, pciutils, gobjectIntrospection }:
+
+stdenv.mkDerivation rec {
+  pname = "switchboard-plug-about";
+  version = "2.5.1";
+
+  name = "${pname}-${version}";
+
+  src = fetchFromGitHub {
+    owner = "elementary";
+    repo = pname;
+    rev = version;
+    sha256 = "1vwjz5gj1c30sb05wi4k2rk5l9z3sk255m0ar1das58sxfr5fyqj";
+  };
+
+  passthru = {
+    updateScript = elementary.updateScript {
+      repoName = pname;
+    };
+  };
+
+  nativeBuildInputs = [
+    gobjectIntrospection
+    meson
+    ninja
+    pkgconfig
+    vala
+  ];
+
+  buildInputs = [
+    granite
+    gtk3
+    libgee
+    switchboard
+  ];
+
+  patches = [
+    (substituteAll {
+      src = ./lspci-path.patch;
+      pciutils = "${pciutils}";
+    })
+    ./remove-update-button.patch
```

Comment:
Maybe we could convince them to add a meson option?

Reply:
An option for the distro logo would be nice too.